CREATING_TOXICS.md

```diff
@@ -0,0 +1,137 @@
+# Creating custom toxics
+
+Creating a toxic is done by implementing the `Toxic` interface:
```

Comment:
Add something here about the fact they have to be merged (or maintained) in a fork currently, but for the most part the toxics built in are all we use, so we don't think this is a huge concern. You can give some suggestions of protocol-specific toxics, e.g. a Redis toxic where you can drop percentages of reads and writes. However, with a note that it may not be terribly useful but for edge-cases as your worst case are still those provided by the latency toxic and downtime.